src/main/java/net/minestom/server/utils/mojang/MojangUtils.java

```diff
@@ -1,32 +1,25 @@
 package net.minestom.server.utils.mojang;
 
-import com.github.benmanes.caffeine.cache.Cache;
-import com.github.benmanes.caffeine.cache.Caffeine;
 import com.google.gson.JsonObject;
 import com.google.gson.JsonParser;
-import net.minestom.server.MinecraftServer;
 import net.minestom.server.utils.url.URLUtils;
 import org.jetbrains.annotations.Blocking;
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
 import java.io.IOException;
 import java.util.UUID;
-import java.util.concurrent.TimeUnit;
 
 /**
  * Utils class using mojang API.
  */
 public final class MojangUtils {
     private static final String FROM_UUID_URL = "https://sessionserver.mojang.com/session/minecraft/profile/%s?unsigned=false";
     private static final String FROM_USERNAME_URL = "https://api.mojang.com/users/profiles/minecraft/%s";
```

Comment:
why is this change here? it shouldn't be here

Reply:
artifact of my obvious skill issue with regards to merging. i'll fix that